test/extended/ldap_groups.sh

```diff
@@ -229,6 +229,13 @@ for (( i=0; i<${#schema[@]}; i++ )); do
     popd > /dev/null
 done
 
+# special test for RFC2307
+pushd ${BASETMPDIR}/rfc2307 > /dev/null
+echo -e "\tTEST: Sync groups from LDAP server, tolerating errors"
+oadm groups sync --sync-config=sync-config-tolerating.yaml --confirm
```

Comment:
any way to capture the output and make sure the lookup and out of scope error got logged?